workflow/rules/trimming.smk

```diff
@@ -53,49 +53,60 @@ rule cutadapt_se:
     wrapper:
         "0.42.0/bio/cutadapt/se"
 
-
-rule trimmomatic_se:
+rule pipe_ptrimmer_se:
     input:
         "results/trimmed/adapters/{sample}/{unit}.single.fastq.gz"
+    output:
+        pipe("results/trimmed/adapters/{sample}/{unit}.single_R1.fastq.gz")
+    log:
+        "logs/pipe-fastqs/ptrimmer/{sample}-{unit}.single.log"
+    threads: 0 # this does not need CPU
+    shell:
+        "cat {input} > {output} 2> {log}"
+
+
+rule ptrimmer_se:
+    input:
+        "results/trimmed/adapters/{sample}/{unit}.single_R1.fastq.gz"
```

Comment:
Are you sure that it complains about the file name in the single end case?

Reply:
Currently not, but if the input for `ptrimmer_se` becomes `"results/trimmed/adapters/{sample}/{unit}_R1.fastq.gz"` both rules `cutadapt_pe` and `cutadapt_se` need to produce the same output.

Reply:
Changing input to `_R1` for single-end results in the following error:
```
AmbiguousRuleException:
Rules cutadapt_se and cutadapt_pe are ambiguous for the file results/trimmed/adapters/a/lane1_R1.fastq.gz.
Consider starting rule output with a unique prefix, constrain your wildcards, or use the ruleorder directive.
Wildcards:
	cutadapt_se: sample=a,unit=lane1
	cutadapt_pe: sample=a,unit=lane1
Expected input files:
	cutadapt_se: pipe/cutadapt/a/lane1.fq1.fastq pipe/cutadapt/a/lane1.fq2.fastq
	cutadapt_pe: pipe/cutadapt/a/lane1.fq1.fastq pipe/cutadapt/a/lane1.fq2.fastq
Expected output files:
	cutadapt_se: results/trimmed/adapters/a/lane1_R1.fastq.gz results/trimmed/adapters/a/lane1.single.qc.txt
	cutadapt_pe: results/trimmed/adapters/a/lane1_R1.fastq.gz results/trimmed/adapters/a/lane1_R2.fastq.gz results/trimmed/adapters/a/lane1.paired.qc.txt
```

Reply:
Add a ruleorder statement: `ruleorder: cutadapt_pe > cutadapt_se`.